### DISCUSSION TITLE
Make Default List Limit a text field in stead of a drop down

### DISCUSSION THREAD

**Discussion by stimpsonjcat** [joomla/joomla-cms#46997] [2. Cold Features]
### Steps to reproduce the issue

Default List Limit in the System Settings has a drop down menu with fixed numbers in steps of 5. It would be beter to have a textbox, like the Article Settings, where you can set any number of article count. Components that rely on the Default List Limit, like the list of articles with a certain tag (com-tags / view tag), can't be set to the same article count as the blog or list view of articles. 

### Expected result

If I create a template with a blog-like view of tagged articles, I want it to look exactly like the blog view of articles.

### Actual result

If you've set the articles blog view count to 12, the tag view is either 15 or 10.
Compare: [https://www.ugenda.nl/interviews](https://www.ugenda.nl/interviews "https://www.ugenda.nl/interviews")
With: [https://www.ugenda.nl/component/tags/tag/de-muzikale-inspiratie-van](https://www.ugenda.nl/component/tags/tag/de-muzikale-inspiratie-van "https://www.ugenda.nl/component/tags/tag/de-muzikale-inspiratie-van")

### System information (as much as possible)

Joomla 4.3.2

### Additional comments

If you have created a template with a blog view in 4 columns for both articles and for com-tags with view tag, the tag view article count can't be set to a number that is a multiple of 4 (except if you set the count to 20 or 100.)

**Comment by brianteeman** [comment]
what you should be doing is to createa menu item for the tags

**Comment by bembelimen** [comment]
You can adjust the filter by setting the elements in the dropdown (either changing the form field in your override or having it directly in your new XML if you do a full new type):
```limits="4,8,20,40,120,..."```

Parameters you could use:
![grafik](https://github.com/joomla/joomla-cms/assets/1229869/847a3f3c-8863-4f58-afda-9722b424538f)

**Comment by stimpsonjcat** [comment]
@brianteeman I can create a menu item with a list go tagged items, which requirers me to select specific tags. Beside, the list limit settings here is also a dropdown with preconfigured counts.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41110">issues.joomla.org/tracker/joomla-cms/41110</a>.</sub>

**Comment by stimpsonjcat** [comment]
@bembelimen This requires an override. I would rather see it as a core feature. 
Is there a reason the setting is a preconfigured drop down?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/41110">issues.joomla.org/tracker/joomla-cms/41110</a>.</sub>
